userspace/chisel/chisel.cpp

```diff
@@ -1170,33 +1170,42 @@ bool sinsp_chisel::openfile(string filename, OUT ifstream* is)
 	return false;
 }
 
-void sinsp_chisel::load(string cmdstr)
+void sinsp_chisel::load(string cmdstr, bool is_file)
 {
-	m_filename = cmdstr;
-	trim(cmdstr);
+	if (is_file) {
+		m_filename = cmdstr;
+		trim(cmdstr);
```

Comment:
Why are we trimming `cmdstr` instead of `m_filename` here? 
It seems like a bug (that was already there of course :D )

WDYT?

Reply:
That is an excellent question, I guess no one ever tried to set a path that was not trimmed already. Let me change it so we actually trim before assigning.